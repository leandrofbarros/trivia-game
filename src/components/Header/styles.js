import { StyleSheet } from 'react-native'

export default StyleSheet.create({
  container: {
    marginTop: 50,
    minHeight:70
  },
  title: {
    fontSize: 28,
    fontWeight: '700',
    textAlign: 'center',
  }
})
