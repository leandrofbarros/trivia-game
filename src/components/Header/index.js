// @flow

import React from 'react'
import { View, Text } from 'react-native'
import styles from './styles'

type Props = {
  title: string,
  subtitle: string
}

const Header = ({ title, subtitle }: Props) => (
  <View style={styles.container}>
    <Text style={styles.title}>{title}</Text>
    <Text style={styles.title}>{subtitle}</Text>
  </View>
)

export default Header
