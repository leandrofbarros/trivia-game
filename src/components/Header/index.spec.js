// @flow

import React from 'react'
import renderer from 'react-test-renderer'

import Header from './index'

describe('#Button', () => {
  it('Renders correctly', () => {
    const component = (
      <Header
        title="Test Title"
        subtitle="Test subtitle"
      />
    )

    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})