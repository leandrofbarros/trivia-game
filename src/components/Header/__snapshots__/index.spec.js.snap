// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Button Renders correctly 1`] = `
<View
  style={
    Object {
      "marginTop": 50,
      "minHeight": 70,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 28,
        "fontWeight": "700",
        "textAlign": "center",
      }
    }
  >
    Test Title
  </Text>
  <Text
    style={
      Object {
        "fontSize": 28,
        "fontWeight": "700",
        "textAlign": "center",
      }
    }
  >
    Test subtitle
  </Text>
</View>
`;
